Cubical/Core/Primitives.agda

```diff
@@ -140,25 +140,31 @@ ouc = primSubOut
 -- When calling "hcomp A φ u a" Agda makes sure that "a" agrees with "u i0" on "φ".
 -- hcomp : ∀ {ℓ} {A : Set ℓ} {φ : I} (u : I → Partial A φ) (a : A) → A
 
+private
+  variable
+    ℓ  : Level
+    ℓ′ : I → Level
+    A′ : Set ℓ
+    φ : I
```

Comment:
I would put A' and phi back below so that the rules read exactly like the typing rule from the papers. Also, why is it A' and not A?

Reply:
Because `A` is already used a lot in the rest of the file and they're not `Set l`.

Reply:
Ok, let's put the types back so that the definitions later look like the typing rules in our papers. Maybe one should format the types of the Kan operations like in https://www.cl.cam.ac.uk/~rio22/agda/cubical-topos/fibrations.html#3142